### ISSUE TITLE
'TypeError: (0 , _reduxPersistSensitiveStorage.default) is not a function' while running test cases

### ISSUE THREAD

**Issue by ghost** [CodingZeal/redux-persist-sensitive-storage#37]
Used this redux-persist-sensitive-storage library and faced the above issue while running the test cases. 

Tests suite failed to run 
`TypeError: (0 , _reduxPersistSensitiveStorage.default) is not a function`

Using @testing-library/react-native

How to resolve this issue? Please let me know.
Thank you

**Comment by davortrifunov** [comment]
Did you manage to find a solution to this one?

**Comment by ghost** [comment]
Yes, the issue was resolved by adding `redux-persist-sensitive-storage` to the transformIgnorePatterns in the `jest.config.js` file.
```
transformIgnorePatterns: [
    'node_modules/(?!(jest-)?@?react-native|redux-persist-sensitive-storage)'
  ],
```
